app/assets/javascripts/admin/addon/components/theme-settings-editor.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="settings-editor settings">
+  <div class="setting">
```

Comment:
Same as above

```suggestion
  <div>
```

Can we drop the extra div then?

Reply:
cold